app/konnect/centralized-consumers.md

```diff
@@ -0,0 +1,129 @@
+
+## How Centralized Consumers work
+
+When you create a centralized Consumer, you must assign it to a realm. A realm groups centralized Consumers around an identity, defined by organizational boundaries, such as a production realm or a development realm. Realms are connected to a [geographic region](/konnect/geo/) in {{site.konnect_short_name}}.
+
+The following diagram shows how you can configure centralized Consumers in realms as well as scope Consumers to a Control Plane:
+
+<!--vale off -->
+{% mermaid %}
+flowchart TD
+A(Consumer: Cruz<br>Consumer Groups: Gold for appointments, Silver for billing)
+B(Consumer: Ira<br>Consumer Groups: Gold for appointments)
+C(Route: /appointments)
+D(Gold tier: 1000 req/60 sec)
+E(Route: /dev)
+F(Consumer: Izumi)
+G(Consumer: Sasha)
+H(Route: /billing)
+I(Silver tier: 10 req/60 sec)
+
+
+subgraph id1 [Prod realm]
+direction LR
+    A
+    B
+end
+subgraph id2 [Dev realm]
+direction LR
+    G
+end
+
+
+id1 --> id3
+id1 --> id4
+id2 --> id5
+
+subgraph id3 [Appointments Control Plane]
+direction LR
+C
+D
+end
+
+subgraph id4 [Billing Control Plane]
+direction LR
+H
+I
+end
+
+subgraph id5 [Dev Control Plane]
+direction LR
+F
+E
+end
+
+style id1 stroke-dasharray:3,rx:10,ry:10
+style id2 stroke-dasharray:3,rx:10,ry:10
+style id3 stroke-dasharray:3,rx:10,ry:10
+style id4 stroke-dasharray:3,rx:10,ry:10
+style id5 stroke-dasharray:3,rx:10,ry:10
+{% endmermaid %}
+<!-- vale on-->
+
+## Create centralized Consumers and Realms 
+
+You can create centralized Consumers using the {{site.konnect_short_name}} API. Only Org Admins have CRUD permissions for centralized Consumers. 
+
+1. Use the following custom {{site.base_gateway}} image when you create a [Data Plane node](/konnect/gateway-manager/data-plane-nodes/):
+   ```
+   kong/kong-gateway-dev
+   ```
+1. Use the `/realms` endpoint to create a realm and optionally associate it with allowed Control Planes and time-to-live values:
+   ```
+   curl -X POST \
+   https://{region}.api.konghq.com/v1/realms \
+   -H "Content-Type: application/json" \
+   -H "Authorization: Bearer $KONNECT_TOKEN" \
+   -d '{
+        "name": "prod",
+        "allowed_control_planes": [
+            "$CONTROL_PLANE_UUID"
+        ],
+        "ttl": 10,
+        "negative_ttl": 10,
+        "consumer_groups": [
+            "$CONSUMER_GROUP"
+        ]
+    }'
+   ```
+   Save the ID of the realm.
+
+   Be sure to replace the following with your own values:
+   * {region}: Region for your {{site.konnect_short_name}} instance.
+   * $KONNECT_TOKEN: Replace with your {{site.konnect_short_name}} personal access token.
+   * $CONTROL_PLANE_UUID: (Optional) Replace with your Control Plane UUID.
+   * `ttl` and `negative_ttl`: (Optional) Time-to-live in minutes of the consumer or negative consumer for this realm in the {{site.base_gateway}} cache.
```

Comment:
What's a `negative consumer`? I don't think that this is what `negative_ttl` means. @zekth can you help us out here?

Reply:
there's no concept of `negative_consumer`. `negative_ttl` represents the TTL of a bad login cache entry. Meaning api key `foobar` is sent to the gateway; gateway reaches the service to authenticate it; if it fails the api key `foobar` will be labelled as non working in gateway cache to the time of the TTL. During that time it won't try to reache the authentication server.